pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "cloudevents>=1.11.0,<2.0.0",
     "numpy>=2.2.2,<3.0.0",
     "mcp>=1.7.1,<2.0.0",
+    "websockets>=15.0.0,<16.0.0",
```

Comment:
this was added bc MCP's websocket client requires it so in the tests i saw MCP tool schema creation fail silently... so went ahead and added a core dep